lib/utils.js

```diff
@@ -147,6 +147,30 @@ function generateLogPrefix(obj, sessionId = null) {
   );
 }
 
+/**
+ * Converts the given object to an integer number if possible
+ *
+ * @param {any} value to be converted
+ * @returns {number?}
+ */
+function convertToInt(value) {
+  switch (typeof value) {
+    case 'number':
+      if (Number.isNaN(value)) {
```

Comment:
this and the below conditions could be simplified using the ternary operator, for example: `return Number.isNaN(value) ? null : value;`